### PR TITLE
issue #74 file location problem

### PR DESCRIPTION
**Pull request by wsredniawa**

### CODE REVIEW

**Comment by ccluri** [comment]
@abukaj can you please review this?

File: figures/kCSD_properties/kCSD1D_test_fig1_and_fig2_v2.py [hunk 1]
```diff
@@ -333,12 +333,13 @@ def main_loop(src_width, total_ele, inpos, lpos, nm, noise=0, srcs=1):
     total_ele = 32
     names = ['lc', 'cv']
     src_width = 0.001
-    noise_lvl = np.linspace(0, 0.5, 10)
+    noises = 3
```
Comment (abukaj) [inline figures/kCSD_properties/kCSD1D_test_fig1_and_fig2_v2.py]: I like this code parametrization, but the name is not the most intuitive. How about:
```suggestion
    noise_lvl_n = 3
```
?

**Comment by abukaj** [comment]
May it wait till 24th of Feb? How urgent is this?

**Comment by ccluri** [comment]
@abukaj Not urgent. Can wait.

**Comment by abukaj** [comment]
There is at least one failing import: https://travis-ci.org/Neuroinflab/kCSD-python/jobs/491655704
@wsredniawa may you take a look at this?

File: kcsd/tests/test_oKCSD.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+try:
+    basestring
+except NameError:
+    basestring = str
+
+
+n_src = 5
+
+class testsKCD(unittest.TestCase):
+    def setUpClass(self):
```
Comment (asiaszmek) [inline kcsd/tests/test_oKCSD.py]: setUpClass as a method name is confusing. Please divide this function into two separate functions testing initialization of oKCSD2D and oKCSD3D and name them accordingly (test_oKCSD2D_initialization and test_oKCSD3D_initialization).

**Comment by asiaszmek** [comment]
I think I am OK with Władek's changes.

File: kcsd/KCSD.py [hunk 3]
```diff
@@ -1066,6 +1065,126 @@ def int_pot_3D(self, xp, yp, zp, x, R, h, basis_func):
...
+                Defaults to 0.
+        Raises
+        ------
+        LinAlgError
+            Could not invert the matrix, try changing the ele_pos slightly
+        KeyError
+            Basis function (src_type) not implemented.
+            See basis_functions.py for available
+        """
+        super().__init__(ele_pos, pots, **kwargs)
```
Comment (asiaszmek) [inline kcsd/KCSD.py]: super(oKCSD2D, self).__init__(ele_pos, pots, **kwargs)

File: kcsd/KCSD.py [hunk 4]
```diff
@@ -1066,6 +1065,126 @@ def int_pot_3D(self, xp, yp, zp, x, R, h, basis_func):
...
```
Comment (asiaszmek) [inline kcsd/KCSD.py]: super(oKCSD3D, self).__init__(ele_pos, pots, **kwargs)

File: kcsd/KCSD.py (hunk 3 above)
Comment (abukaj) [reply kcsd/KCSD.py]: There is a nice feature with the "+- on a stack of papers" widget:
```suggestion
        super(oKCSD2D, self).__init__(ele_pos, pots, **kwargs)
```

File: kcsd/KCSD.py [hunk 5]
```diff
@@ -352,7 +352,7 @@ def suggest_lambda(self):
         u, s, v = svd(self.k_pot)
         print('min lambda', 10**np.round(np.log10(s[-1]), decimals=0))
         print('max lambda', str.format('{0:.4f}', np.std(np.diag(self.k_pot))))
-        return np.logspace(np.log10(s[-1]), np.std(np.diag(self.k_pot)), 20)
+        return np.logspace(np.log10(s[-1]), np.log10(np.std(np.diag(self.k_pot)), 20)
```
Comment (asiaszmek) [inline kcsd/KCSD.py]: Parenthesis missing

**Comment by ccluri** [comment]
@wsredniawa Lets have a teleconference (google hangouts - user name ccluri) about this PR. I am quite confused as to how this is used. Anyone else interested  @asiaszmek  @abukaj  @danek8317 to participate? 

GMT Timezone (< 1Hr of your choosing)
Friday (8th March) - 0900 to 1200
Tuesday (12th March) - 0900 to 1700
Wednesday (13th March) - 1200 to 1700
Thursday (14th March) - 0900 to 1700

File: kcsd/KCSD.py (hunk 5 above)
Comment (abukaj) [reply kcsd/KCSD.py]:
```suggestion
        return np.logspace(np.log10(s[-1]), np.log10(np.std(np.diag(self.k_pot)), 20))
```
?

**Comment by abukaj** [comment]
@ccluri 
> Friday (8th March) - 0900 to 1200
Very likely I will not manage to; have to check the other dates.

**Comment by abukaj** [comment]
> @ccluri
> > Friday (8th March) - 0900 to 1200
> Very likely I will not manage to; have to check the other dates.

Oh, it is UTC - then for sure I am in the lab after 1100.

**Comment by abukaj** [comment]
@ccluri 
Tuesday (12th March) - after 1000 (if there are no issues detected with beta tests of Boar 2.0)
Wednesday and Thursday (13th March) - works for me

**Pull request closed without merging**